catalog/app/containers/NavBar/Help.tsx

```diff
@@ -79,7 +79,7 @@ const syntaxHelpRows: SyntaxHelpNamespace[] = [
         title: 'Object key',
       },
       {
-        example: (s) => <>{s}"research"</>,
+        example: (s) => <>{s}"phase*"</>,
```

Comment:
Shouldn't it be `{s}phase*` without the quotes? Without the quotes works for me on open. With quotes returns no results.